src/main.js

```diff
@@ -72,4 +73,10 @@ if (!window.WebVRConfig.DEFER_INITIALIZATION) {
   window.InitializeWebVRPolyfill = function() {
     new WebVRPolyfill();
   }
+  // Call this if you want to use the shim without the rest of the polyfill.
+  // InitializeWebVRPolyfill() will install the shim automatically when needed,
+  // so this should rarely be used.
+  window.InitializeSpecShim = function() {
```

Comment:
IMO, we should not be adding yet another global for the WebVR Polyfill. see #47; I have a branch of this I've done locally that moves everything to a single `WEBVR_POLYFILL`  global that is exported from `./webvr-polyfll.js` and then from here in `src/main.js`.